spec/latest/core/index.html

```diff
@@ -1239,14 +1239,17 @@ <h3>Templated Links</h3>
         "@type": "Collection",
         "some:operation": {
           "@type": "IriTemplate",
-          "template": "/{id}",
+          "template": "{id}",
           "resolveRelativeTo": "LinkContext",
           "variable": "id",
           "mapping": {...}
         }
       }
       -->
     </pre>
+    
+    <p>When constructed, the URL would effectively become similar to <code>http://api.example.com/an-issue/1234</code>,
```

Comment:
Shouldn't we use `IRI` consistently?

```suggestion
    <p>When constructed, the IRI would effectively become similar to <code>http://api.example.com/an-issue/1234</code>,
```